### ISSUE TITLE
Accessibility—missing skip-links, reader not reading

### ISSUE THREAD

**Issue by jenn-sweet** [Phan717/web-design-2-independent-project#14]
Issue: Missing skip links
Issue: Accessibility not finished at time of viewing
Issue: Alt descriptions not written out at time of viewing

<img width="1266" alt="Screen Shot 2021-12-11 at 7 26 08 PM" src="https://user-images.githubusercontent.com/77341921/145695954-d352c714-cc07-4d9e-ade1-b6383abcf030.png">
